### DISCUSSION TITLE
Do you have plans to add support php 8.4.x for Phalcon 5.8?

### DISCUSSION THREAD

**Discussion by serhii-m-prime** [phalcon/cphalcon#16669] [General]
Ask just in case of:
`pecl/phalcon requires PHP (version >= 8.0.0, version <= 8.3.99), installed version is 8.4.1
No valid packages found
`

**Comment by sinbadxiii** [comment]
> Ask just in case of: `pecl/phalcon requires PHP (version >= 8.0.0, version <= 8.3.99), installed version is 8.4.1 No valid packages found `

yes it will, https://github.com/phalcon/cphalcon/issues/16671
and now we need to wait https://github.com/zephir-lang/zephir/issues/2440

**Reply by serhii-m-prime** [reply]
But here about 6.0, any updates about 5.8?

**Reply by sinbadxiii** [reply]
is not, 
it is excatly cphalcon ext, 
after making support PHP 8.4 from zephir it might be possible to compile phalcon 5.x for PHP 8.4 too

**Comment by HalleyUK** [comment]
Hi there, 
I also have the similar question. My application is currently running on Phalcon 5.4 and Php8.1.
Would be great to know when php8.4 is supported so we can upgrade both Phalcon and php versions this year.
Thank you very much.

**Comment by niden** [comment]
Still working on it guys. Sorry for the delay
